tendrl/commons/utils/alert_utils.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def alert_job_status(curr_value, msg, integration_id=None, cluster_name=None):
+    alert = {}
+    alert['source'] = NS.publisher_id
+    alert['classification'] = 'cluster'
+    alert['pid'] = os.getpid()
+    alert['time_stamp'] = tendrl_now().isoformat()
+    alert['alert_type'] = 'STATUS'
+    severity = "INFO"
+    if curr_value.lower() == "failed":
+        severity = "WARNING"
+    alert['severity'] = severity
+    alert['resource'] = 'job_status'
+    alert['current_value'] = curr_value
+    alert['tags'] = dict(
+        message=msg,
+        integration_id=integration_id or
+        NS.tendrl_context.integration_id,
+        cluster_name=cluster_name or
+        NS.tendrl_context.cluster_name,
+        sds_name=NS.tendrl_context.sds_name,
+        fqdn=NS.node_context.fqdn
+    )
+    alert['node_id'] = NS.node_context.node_id
+    if not NS.node_context.node_id:
+        return
+    logger.log(
+        "notice",
```

Comment:
This shouldnt be "notice" , it should be either "info" if job succeeded or "error" if job failed/timedout

Reply:
@r0h4n I remember if its not `notice` the data goes to `messages/events` only and not to `alerting/clusters/{int-id}`.

Reply:
After discussion with @GowthamShanmugam and looking at api implementation at https://github.com/Tendrl/api/blob/master/app/models/notification.rb, we find that notifications drawer menu in UI gets data from `messages/events` but only messages with `log_priorty = notice` are listed here.

So the changes in this PR remain required. As per @GowthamShanmugam to supress, email alerts for these, we can set `delivered = True` for alert message payload. This has a drawback (but not braking any functionality) that the job success/failure would be listed under Alerts as well. 

@GowthamShanmugam needs a fix in node-agent's alert module to expect another parameter in alert payload to not put the `notice` type message in `/alerting` and it would meant for only notifications. Till this change comes, we would go ahead with parameter `delivered = True` to achieve this.

Will be updating the PR with this soon. Hope this ends the long going discussion around this :) no offences ;)